src/App.tsx

```diff
@@ -1,20 +1,26 @@
 import './App.css';
 import PresentThemeContainer from '@components/PresentThemeContainer';
-import PresentContainer from '@components/PresentContainer';
+import PresentRankingContainer from '@/components/PresentRankingContainer';
 import SelectFriendContainer from '@components/SelectFriendContainer';
 import styled from '@emotion/styled';
+import NavigationBar from './components/NavigationBar';
 
 const StyledTopestDiv = styled.div`
   background-color: ${({ theme }) => theme.palette.gray400};
 `;
-
+const Spacer = styled.div`
+  //NavigationBar가 상단에 fixed될때 다른 컴포넌트를 가리는 문제르 해결하기 위한 공백 공간
+  height: 50px;
+`;
 function App() {
   return (
     <>
       <StyledTopestDiv>
+        <NavigationBar></NavigationBar>
```

Comment:
children이 없다면 self-closing tag를 써도 좋겠네요!
요런 부분은  prettier나 lint 설정으로 자동 적용되게 만들 수도 있어요 👍 

```suggestion
        <NavigationBar />
```